modules/process/pages/managing-dependencies.adoc

```diff
@@ -9,6 +9,11 @@ A dependency is an external module, not provided by the Bonita runtime, that is
 You can have an overview of your project dependencies using the project _Overview_ in a Bonita Studio. From there you can navigate to the _Extensions_ view.
 This view only displays the root dependencies. In most cases, a java dependency has transitive dependencies. Bonita Studio helps you deal with those transitive dependencies by using Apache Maven as a dependency management tool for your Bonita project.
 
+[NOTE]
+====
+If you need to review all the dependencies associated with your project—or just a specific process—simply right-click on the desired element and select Dependency Tree. This will displays a comprehensive Maven dependency tree, detailing every dependency included in your project.
+====
+
 == Configure process dependencies
```

Comment:
We would need another chapter here "Environment-specific dependencies" to tell how to create maven profiles for custom dependencies per environment.

Also, the configuring-a-process.adoc page must be updated to remove obsolete mentions.